### PR TITLE
Update zh-rTW translation

### PR DESCRIPTION
**Pull request by Lafudoci**
Update some translation of new strings for ledger and NFC tag.
@0140454 Could you help me to review the translations?

### CODE REVIEW

**Review by erciccione — APPROVED** [review]
Technically looks OK. Thanks @Lafudoci :)

**Comment by 0140454** [comment]
@Lafudoci @m2049r
Should we insert space between Chinese and English?

Because there are many inconsistencies in the translation, I think we should unify it.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -320,29 +320,29 @@
...
+    <string name="accounts_progress_update">正在更新帳戶</string>
+
+    <string name="toast_ledger_attached">%1$s 已連接</string>
+    <string name="toast_ledger_detached">%1$s 已斷開連接</string>
+
+    <string name="progress_nfc_write">正在寫入NFC標籤</string>
+    <string name="nfc_write_failed">NFC標籤寫入失敗！</string>
+    <string name="nfc_write_successful">NFC標籤寫入成功</string>
+    <string name="nfc_tag_unsupported">此NFC標籤不支援 NDEF 格式！</string>
+    <string name="nfc_tag_size">此NFC標籤可存放 %d 位元組，但尚需 %d！</string>
```
Comment (0140454) [inline app/src/main/res/values-zh-rTW/strings.xml]: @Lafudoci
「尚需」改成「需要」會不會比較好？

我覺得「尚需」有「還需要」的意思，表示還缺少多少 bytes。
但第二個數字看起來應該是總共需要的大小？

Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/strings.xml]: 確實「需要」是比較精確的，感謝指正！

**Comment by Lafudoci** [comment]
@0140454 [Spacing might be better](http://w3c.github.io/clreq/#mixed_text_composition_in_horizontal_writing_mode). And the inconsistencies need to be revised for sure. It seems there is no layout engine handling mixed text, right? @m2049r So I believe we have to manually add spaces to all mixed texts. I could do that if we have consensus.

**Comment by Lafudoci** [comment]
I have unified all the spaces between Eng and zh character. Beside, some minor translation issues were revised.

**Review by erciccione — APPROVED** [review]

**Comment by 0140454** [comment]
@Lafudoci 
I create a new [PR](https://github.com/Lafudoci/xmrwallet/pull/1) to your repository, could you review it?

The main changes are
1. Unifty the space and punctuation marks
2. Adjust the position of line break

**Comment by Lafudoci** [comment]
@0140454 , I have reviewed and merged your revision. Nice job!

**Review by 0140454 — APPROVED** [review]

**Pull request merged**